### DISCUSSION TITLE
OAP is running in 'no-init' mode, waiting... retry 3s later.

### DISCUSSION THREAD

**Discussion by jiangxiaobai** [apache/skywalking#9093] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Helm Chart (apache/skywalking-kubernetes)

### What happened

1、prepare an elasticsearch cluster 7.3
2、install skywalking 9.0 via helm3, use es installed in the step 1
3、wait es-init to complete and oap/ui to get running，everything is ok and not found any errors in logs
4、kubectl scale --replicas=0 deployment/skywalking-oap
5、kubectl scale --replicas=2 deployment/skywalking-oap
oap server shows the following logs: [main] INFO  [] - table: alarm_record does not exist. OAP is running in 'no-init' mode, waiting... retry 3s later.

### What you expected to happen

skywalking-oap starts normally,and I want to know how to run the init-task again in the helm charts.

### How to reproduce

kubectl scale --replicas=0 deployment/skywalking-oap

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
I think you are not following the steps you described. Such as 
1. Do you clear the ElasticSearch manually?
2. How long do you invoke (5) after step (4)? More than one day?

**Reply by jiangxiaobai** [reply]
1、 no
2、  More than one day

**Reply by jiangxiaobai** [reply]
How to run the init-task again in the helm charts,The page(https://github.com/apache/skywalking/blob/master/.github/workflows/e2e.istio.yaml#L88) you provided not found

**Reply by wu-sheng** [reply]
One than one day, then there was a bug existing, AFAIK only fixed in 9.1.0(next release).

**Reply by wuzhihui1123** [reply]
> One than one day, then there was a bug existing, AFAIK only fixed in 9.1.0(next release).

i have the same problem,  my skywalking version is 9.1.0
Everything works fine the first time it starts up, and it will appear error after running for a while, and always restart, error message:
`
table: log does not exist. OAP is running in 'no-init' mode, waiting... retry 3s later
`

**Reply by sivern** [reply]
I'm version 8.9.1, and this problem happened a few months after normal operation, and I've observed that the index template still exists

**Reply by djsxianglei** [reply]
@wuzhihui1123 Excuse me, has this problem been solved?i have the same problem, my skywalking version is 9.2.0

**Comment by jibingjie2** [comment]
Is this problem solved?

**Reply by Superskyyy** [reply]
@jibingjie2 see below

**Comment by wu-sheng** [comment, marked as answer]
The fix had been posted for a while, https://github.com/apache/skywalking/discussions/9634

**Comment by sivern** [comment]
I solved it by modifying the plugin myself




学习机
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apache/skywalking"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月9日(星期四) 下午3:28
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apache/skywalking] OAP is running in 'no-init' mode, waiting... retry 3s later. (Discussion #9093)





 
@wuzhihui1123 Excuse me, has this problem been solved?i have the same problem, my skywalking version is 9.2.0
